frontend/src/lib/components/CompareFilter/compareFilterLogic.test.ts

```diff
@@ -1,17 +1,20 @@
 import { compareFilterLogic } from 'lib/components/CompareFilter/compareFilterLogic'
-import { initKeaTestLogic } from '~/test/init'
+import { initKeaTests } from '~/test/init'
 import { router } from 'kea-router'
 import { urls } from 'scenes/urls'
 import { expectLogic } from 'kea-test-utils'
-import { InsightShortId, InsightType } from '~/types'
+import { InsightLogicProps, InsightShortId, InsightType } from '~/types'
+import { insightLogic } from 'scenes/insights/insightLogic'
 
 describe('compareFilterLogic', () => {
     let logic: ReturnType<typeof compareFilterLogic.build>
 
-    initKeaTestLogic({
-        logic: compareFilterLogic,
-        props: {},
-        onLogic: (l) => (logic = l),
+    beforeEach(() => {
+        initKeaTests()
```

Comment:
I like the reduction in magic from `initKeaTestLogic` to this `beforeEach` approach